dvc/project.py

```diff
@@ -113,13 +122,19 @@ def init(root_dir=os.curdir, no_scm=False, force=False):
         dvc_dir = os.path.join(root_dir, Project.DVC_DIR)
         scm = SCM(root_dir)
         if type(scm) == Base and not no_scm:
-            msg = "{} is not tracked by any supported scm tool(e.g. git)."
-            raise InitError(msg.format(root_dir))
+            raise InitError(
+                "{project} is not tracked by any supported scm tool"
+                " (e.g. git). Use '--no-scm' if you don't want to use any scm."
+                .format(project=root_dir)
```

Comment:
just curious - does this with unicode in Python 2? do we import a future for that?

Reply:
Good question, @shcheklein ! I'm afraid that it doesn't

Reply:
Great catch! Created https://github.com/iterative/dvc/issues/1512 . Need to go through our code base and see that we are working with unicode correctly and add some tests for it(e.g. init in a directory which has unicode characters).